pkg/chrootarchive/diff_windows.go

```diff
@@ -14,6 +15,12 @@ import (
 // contents of the layer.
 func applyLayerHandler(dest string, layer archive.Reader, decompress bool) (size int64, err error) {
 	dest = filepath.Clean(dest)
+
+	// Ensure it is a Windows-style volume path
```

Comment:
@swernli Just to make sure: it's the second time you write `\?\` but in the code it says `\\?\`, is the code correct?